### ISSUE TITLE
Cleaning "Windows Event Logs" with CCleaner causes blank usvfs logs and no mods loaded.

### ISSUE THREAD

**Issue by Al12rs** [ModOrganizer2/modorganizer#366]
# The problem

After running the cleaning function of ccleaner with the Windows Events Logs selected Mo2 will no longer virtualize the mods and usvfs logs result blank. 
![image](https://user-images.githubusercontent.com/26797547/40201516-8fe5749e-5a1f-11e8-885b-a94e4a371d39.png)

Rebooting the PC fixes the problem.

# Environment

* Mod Organizer Version that exhibits the issue:2.1.3
* Last Mod Organizer Verison that did not exhibit the issue (if applicable):
* Desktop OS/version used to run Mod Organizer:Win10

**Comment by hylandnp** [comment]
I've experienced the same issue virtualising the mods. However I had to do more. 

Disabling system monitoring and active monitoring in CCleaner's options menu *then* restarting my PC fixed it for me.

![untitled](https://user-images.githubusercontent.com/3501643/42409503-056779e4-81d3-11e8-9866-4123647bd2ea.png)

I think when CCleaner monitoring is enabled it interferes somehow with Windows Event Log system.
